src/java/com/wolfssl/provider/jsse/WolfSSLEngineHelper.java

```diff
@@ -165,8 +165,7 @@ protected WolfSSLEngineHelper(WolfSSLSession ssl, WolfSSLAuthStore store,
             WolfSSLParameters params, int port, InetAddress peerAddr)
             throws WolfSSLException {
 
-        if (params == null || ssl == null || store == null ||
-            peerAddr == null) {
+        if (params == null || ssl == null || store == null || peerAddr == null || port < 0) {
```

Comment:
Where possible, we still try to conform to our 80-character max line length in the Java wrappers.  Can you wrap this line around?  Looks just a little too long.